shaders/src/dithering.fs

```diff
@@ -53,10 +52,10 @@ vec4 Dither_InterleavedGradientNoise(vec4 rgba) {
     return rgba + vec4(noise / 255.0);
 }
 
-vec4 Dither_TriangleNoise(vec4 rgba) {
+vec4 Dither_TriangleNoise(vec4 rgba, const highp float temporalNoise01) {
```

Comment:
Why is it called 01?

Reply:
it's expected to be between 0 and 1 values. (e.g. as opposed to [-1,1] or [-0.5, 0.5]